package/yast2-nfs-client.spec

```diff
@@ -41,6 +41,12 @@ Requires:       yast2-nfs-common >= 2.24.0
 # showmount, #150382, #286300
 Recommends:     nfs-client
 
+# Unfortunately we cannot move this to macros.yast...
```

Comment:
This is what we found last time: https://github.com/openSUSE/obs-build/blob/5f04f59b27bc1fbd1dfa908c5287657d306fe19e/Build/Rpm.pm#L194